src/res/translation/translation_zh_CN.ts

```diff
@@ -0,0 +1,3072 @@
+        <source>A&amp;dvanced Settings...</source>
+        <translation>高级选项(&amp;D)...</translation>
+    </message>
+    <message>
+        <location filename="../../clientdlg.cpp" line="305"/>
+        <source>&amp;Edit</source>
+        <translation>编辑(&amp;E)</translation>
+    </message>
+    <message>
+        <location filename="../../clientdlg.cpp" line="311"/>
+        <source>Sort Users by &amp;Name</source>
+        <translation>根据名称排序用户(&amp;N)</translation>
+    </message>
+    <message>
+        <location filename="../../clientdlg.cpp" line="313"/>
+        <source>Sort Users by &amp;Instrument</source>
+        <translation>根据乐器排序用户(&amp;I)</translation>
+    </message>
+    <message>
+        <location filename="../../clientdlg.cpp" line="319"/>
+        <source>Sort Users by &amp;Group</source>
+        <translation>根据分组排序用户(&amp;G)</translation>
+    </message>
+    <message>
+        <location filename="../../clientdlg.cpp" line="702"/>
+        <source>Directory Server</source>
+        <translation>目录服务器</translation>
+    </message>
+    <message>
+        <location filename="../../clientdlg.cpp" line="747"/>
+        <location filename="../../clientdlg.cpp" line="759"/>
+        <source>Select Channel Setup File</source>
+        <translation>选择通道配置文件</translation>
+    </message>
+    <message>
+        <location filename="../../clientdlg.cpp" line="896"/>
+        <source>user</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../../clientdlg.cpp" line="900"/>
+        <source>users</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../../clientdlg.cpp" line="935"/>
+        <source>Connect</source>
+        <translation>连接</translation>
+    </message>
+    <message>
+        <location filename="../../clientdlg.cpp" line="947"/>
+        <source>Settings</source>
+        <translation>选项</translation>
+    </message>
+    <message>
+        <location filename="../../clientdlg.cpp" line="957"/>
+        <source>Chat</source>
+        <translation>聊天</translation>
+    </message>
+    <message>
+        <location filename="../../clientdlg.cpp" line="1033"/>
+        <source>Enable feedback detection</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../../clientdlg.cpp" line="1036"/>
+        <source>Audio feedback or loud signal detected.
+
+We muted your channel and activated &apos;Mute Myself&apos;. Please solve the feedback issue first and unmute yourself afterwards.</source>
+        <translation>Feedback audio o segnale forte rilevato.
+
+E&apos; stato disattivato l&apos;audio del tuo canale ed inserito il &quot;Disattiva Inputt&quot;. Si prega di risolvere prima il problema dei larsen e dei ritorni e successivamente riattivare l&apos;audio.</translation>
+    </message>
+    <message>
```

Comment:
Something seems to be wrong here...

Reply:
Yes that need to be fixed. Thanks for pointing that out.